src/SFA.DAS.ApprenticePortal.SharedUi/Identity/IdentityClaims.cs

```diff
@@ -12,7 +12,8 @@ public static class IdentityClaims
         public const string FamilyName = "family_name";
 
         public static Claim? ApprenticeIdClaim(this ClaimsPrincipal user)
-            => user.Claims.FirstOrDefault(c => c.Type == ApprenticeId);
+            => user.Claims.FirstOrDefault(c => c.Type == ApprenticeId)
+            ?? user.Claims.FirstOrDefault(c => c.Type == "registration_id");
```

Comment:
Is registration_id even present now, Thought we had removed all refs to it

Reply:
Yes, so did I.  But I saw some log errors complaining that it couldn't find `apprentice_id`.  I suspect that an old version of the login service was deployed in that environment.